bigchaindb/web/views/transactions.py

```diff
@@ -24,6 +26,8 @@
 from bigchaindb.models import Transaction
 from bigchaindb.web.views.base import make_error
 
+logger = logging.getLogger(__name__)
```

Comment:
It is there so that it is ready for future use, as we should be logging more information in general as per #663.